cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -197,26 +198,27 @@ func generateInitramfsMounts() (err error) {
 
 // generateMountsMode* is called multiple times from initramfs until it
 // no longer generates more mount points and just returns an empty output.
-func generateMountsModeInstall(mst *initramfsMountsState) error {
+func generateMountsModeInstall(mst *initramfsMountsState) (gadget.Model, error) {
```

Comment:
I wonder if instead mst should have a verifiedModel field and a SetVerifiedModel method, then the caller should check and panic if verifiedModel is not set?

Reply:
I pushed an updated with this suggestion, looks quite nice with that, so thank you :)